docs/administration/back_office/back_office_elements/reusable_components.md

```diff
@@ -42,6 +42,85 @@ The alert component has the following properties:
 } only %}
 ```
 
+## Details
+
+The details component consists of the following blocks:
+
+- `details_header`
+
+- `details_items`
+
+Variables:
+
+|Name|Type|Values|
+|----|----|-----------|
+|`headline`|string|
+|`headline_items`|array|
+|`view_mode`|string|`vertical`, default set to `''`|
+|`items`|array|`label`, `content_raw`, `content`|
+
+`headline` and `headline_items` are variables for `@ibexadesign/ui/component/table/table_header.html.twig`
+
+## Modal
+
+The modal component consist of the following blocks:
+
+```html+twig
+{% block dialog %}
+	{% block content_before %}
+	{% block content %}
+		{% block header %}
+		{% block subheader %}
+		{% block body %}
+		{% block footer %}
+	{% block content_after %}
+```
+Variables:
+
+|Name|Type|Values|
+|----|----|-----------|
+|`size`|string|`small`, `large`, `extra-large`, default set to: `''`|
+|`subtitle`|string|default set to `false`|
+|`no_header`|boolean|default set to `false`|
+|`no_header_border`|boolean|default set to `false`|
+|`class`|string|default `''`|
+|`tabindex`|string|default `-1`|
+|`role`|string|default `dialog`|
+|`id`|string||
+|`has_static_backdrop`|boolean|default set to `false`|
+
+`attr`
```

Comment:
If we want to be precise, it would probably be good to state that `attr` and other `attr_*` hold all HTML attributes rendered on their respective elements, but some are defaulted or modified.

But maybe it is stated in another part of the documentation. :)